Resources/Storage/XROOTStorage.py

```diff
@@ -1597,3 +1598,32 @@ def getCurrentURL( self, fileName ):
     cwdUrl = result['Value']
     fullUrl = '%s/%s' % ( cwdUrl, fileName )
     return S_OK( fullUrl )
+
+  def constructURLFromLFN( self, lfn, withWSUrl = False ):
```

Comment:
a) svcClass from cern KB https://cern.service-now.com/service-portal/article.do?n=KB0000953&s=castor%20xrdcp
If it is standard I don't actually know. But as SpaceToken was not used in XRoot Before it shouldn't affect anybody else, and I can add some remarks

b) I had to make some changes in the constructURLFromLFN, function to add svcClass, so I can't use super.
Or do you mean call super and then add svcClass to the result?